ste/mgr-JobMgr.go

```diff
@@ -810,15 +865,20 @@ func (jm *jobMgr) RequestTuneSlowly() {
 func (jm *jobMgr) scheduleJobParts() {
 	startedPoolSizer := false
 	for {
-		jobPart := <-jm.xferChannels.partsChannel
+		select {
+		case jobPart := <-jm.xferChannels.partsChannel:
 
-		if !startedPoolSizer {
-			// spin up a GR to co-ordinate dynamic sizing of the main pool
-			// It will automatically spin up the right number of chunk processors
-			go jm.poolSizer()
-			startedPoolSizer = true
+			if !startedPoolSizer {
+				// spin up a GR to co-ordinate dynamic sizing of the main pool
+				// It will automatically spin up the right number of chunk processors
+				go jm.poolSizer()
+				startedPoolSizer = true
+			}
+			jobPart.ScheduleTransfers(jm.Context())
+
+		case <-jm.Context().Done():
```

Comment:
nit, to be consistent with other place let's make this the first case
Also, instead of having a PoolDestroy() which we call explicitly, I think it makes sense to signal the poolSizer to stop from here only (since from only we start the poolSize and don't have a separate PoolCreate() function.

Reply:
done